library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+    * EOG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Electrooculogram (EOG) channel findings. (Source: Beniczky ea 2017, Section 13.)]</nowiki>
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Respiration-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Respiration channel findings involve the observations and characteristics associated with the recording of respiratory signals.]</nowiki>
-        ** Respiration-oxygen-saturation {requireChild}
+    * Respiration-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [Findings in respiration sensors. (Source: Beniczky ea 2017, Section 13, Table 16.)]</nowiki>
+        ** Oxygen-saturation <nowiki> {requireChild}[Percentage + free text. (Source: Beniczky ea 2013, Table 9; Beniczky ea 2017, Table 16.)] </nowiki>
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text. (Source: Beniczky ea 2017, Table 16.)]</nowiki>
+        ** Apnea <nowiki>[Duration (range in seconds). (Source: Beniczky ea 2013, Table 9.)]</nowiki>
             *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
-        ** Respiration-feature
-            *** Apnoe-respiration <nowiki>[Add duration (range in seconds) and comments in free text.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Hypopnea-respiration <nowiki>[Add duration (range in seconds) and comments in free text]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Apnea-hypopnea-index-respiration <nowiki> {suggestedTag=Frequency} [Events/h.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Periodic-respiration
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Tachypnea-respiration <nowiki> {suggestedTag=Frequency} [Cycles/min.]</nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Other-respiration-feature {requireChild}
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * ECG-channel-finding <nowiki> {suggestedTag=Finding-significance-to-recording} [ECG channel findings include observations and characteristics associated with the Electrocardiogram recording.]</nowiki>
-        ** ECG-QT-period {requireChild}
+        ** Hypopnea <nowiki>[Duration (range in seconds). (Source: Beniczky ea 2013, Table 9.)]</nowiki>
+            *** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
+        ** Apnea-hypopnea-index <nowiki> {suggestedTag=Frequency} [Events/hour. (Source: Beniczky ea 2013, Table 9.)]</nowiki>
```

Comment:
Suggest description of:

"Events/hour as calculated by dividing the number of apnoea and hypopnoea events by the number of hours of sleep. (Source: Wikipedia. Source: Beniczky ea 2013, Table 9.)"